backend/modules/professor/routers/professor_grading.py

```
@@ -209,7 +213,7 @@ async def get_grade_history(
         # Enforce authorization unconditionally - missing cohort linkage results in denied access
         cohort = instance.cohort_assignment.cohort if instance.cohort_assignment else None
         if cohort is None or cohort.created_by != current_user.id:
-            raise HTTPException(status_code=403, detail="Access denied")
+                raise HTTPException(status_code=403, detail="Access denied")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)